docs/architecture/ADR-001-uToken-interest.md

```diff
@@ -0,0 +1,92 @@
+# ADR 001: uToken Interest
+
+## Changelog
+
+- September 10, 2021: Initial draft (@toteki)
+- Jul 2, 2022: Cleanup and simplify (@toteki)
+
+## Status
+
+Implemented
+
+## Context
+
+One of the base functions of the Umee universal capital facility is to allow liquidity providers to deposit assets and earn interest on their deposits.
+
+From section 2.1 of the [Umee Whitepaper](https://umee.cc/umee-whitepaper/):
+
+> Upon deposit of assets into the Asset Facilities, users will receive an amount of tokens called uTokens that map 1:1 with the asset deposited. uTokens are initially minted on Umee and can bridge over to Ethereum as ERC20 tokens.
+>
+> The balance of uTokens grows over time by the underlying interest rate applied to the deposits. uTokens will employ an interest stream mechanism which means that a balance of uTokens will constantly generate income even when split or transferred.
+
+Interest on uTokens should function in at least the following scenarios:
```

Comment:
Maybe we should call them `u-Tokens`. WDYT?

Reply:
`uTokens` seems simpler to me. It also matches the [whitepaper](https://www.umee.cc/umee-whitepaper.pdf) - see section 2.1 for example.

Reply:
OK, won't block here, but my preference is to stick with what we have in the code - the denom we will show to the users. Maybe we can ask others if they have a preference before merging this? I think it's an important decision, and we will need to stick with it in all other documentation pieces.

Reply:
We can discuss it on the Tues call or at convenience on slack. This PR doesn't block anything else so it's no rush.

My current approach is to use the word "uTokens" to refer to the general category, and specifics like `u/UMEE` and `u/ATOM` in example scenarios and code comments.